apps/root/src/pages/home/components/earn-delayed-withdrawals/index.tsx

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import Address from '@common/components/address';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { formatUsdAmount } from '@common/utils/currency';
+import { getDelayedWithdrawals } from '@common/utils/earn/parsing';
+import { DelayedWithdrawalPositions, DelayedWithdrawalStatus, EarnPosition } from 'common-types';
+import {
+  AnimatedChevronRightIcon,
+  colors,
+  ContainerBox,
+  DividerBorder1,
+  Grid,
+  InfoCircleIcon,
+  Skeleton,
+  TickCircleIcon,
+  TimerIcon,
+  Tooltip,
+  Typography,
+} from 'ui-library';
+import usePushToHistory from '@hooks/usePushToHistory';
+import styled from 'styled-components';
+
+const StyledItemContainer = styled(ContainerBox).attrs({
+  justifyContent: 'space-between',
+  alignItems: 'center',
+})`
+  ${({ theme: { palette, spacing } }) => `
+    padding: ${spacing(2)};
```

Comment:
padding should actually be 12px, check the figma there are 2 paddings involved, one of 8px and one of 4px :D

Reply:
🦅